pkg/reconciler/openshift/tektonaddon/tektonaddon.go

```diff
@@ -416,41 +410,27 @@ func createInstallerSet(ctx context.Context, oc clientset.Interface, ta *v1alpha
 		return err
 	}
 
-	is := makeInstallerSet(ta, manifest, installerSetPrefix, releaseVersion, specHash)
-
-	createdIs, err := oc.OperatorV1alpha1().TektonInstallerSets().
-		Create(ctx, is, metav1.CreateOptions{})
-	if err != nil {
-		return err
-	}
-
-	if len(ta.Status.AddonsInstallerSet) == 0 {
-		ta.Status.AddonsInstallerSet = map[string]string{}
-	}
-
-	// Update the status of addon with created installerSet name
-	ta.Status.AddonsInstallerSet[component] = createdIs.Name
-	ta.Status.SetVersion(releaseVersion)
+	is := makeInstallerSet(ta, manifest, installerSetPrefix, releaseVersion, component, specHash)
 
-	_, err = oc.OperatorV1alpha1().TektonAddons().
-		UpdateStatus(ctx, ta, metav1.UpdateOptions{})
-	if err != nil {
+	if _, err := oc.OperatorV1alpha1().TektonInstallerSets().
+		Create(ctx, is, metav1.CreateOptions{}); err != nil {
 		return err
 	}
 
 	return nil
 }
 
-func makeInstallerSet(ta *v1alpha1.TektonAddon, manifest mf.Manifest, prefix, releaseVersion, specHash string) *v1alpha1.TektonInstallerSet {
+func makeInstallerSet(ta *v1alpha1.TektonAddon, manifest mf.Manifest, prefix, releaseVersion, component, specHash string) *v1alpha1.TektonInstallerSet {
 	ownerRef := *metav1.NewControllerRef(ta, ta.GetGroupVersionKind())
 	return &v1alpha1.TektonInstallerSet{
 		ObjectMeta: metav1.ObjectMeta{
 			GenerateName: fmt.Sprintf("%s-", prefix),
 			Labels: map[string]string{
-				tektoninstallerset.CreatedByKey: createdByValue,
+				tektoninstallerset.CreatedByKey:      tektoninstallerset.CreatedByValue,
```

Comment:
`tektoninstallerset.CreatedByValue` this definitely a reconciler specific item, which has to live in the reconciler which is using this (for example pipelines, addons, triggers..)(

Reply:
what i meant is `tektoninstallerset.CreatedByValue` sounds odd, i was expecting just `CreatedByValue` (as CreatedByValue is defined in addons. 

In other words, the value part of this annotation comes from the entity who creates the installerset. So referencing it as `tektoninstallerset.` sounds odd.